core/server/src/test/java/alluxio/master/file/PermissionCheckerTest.java

```diff
@@ -194,6 +194,14 @@ public void before() throws Exception {
     mPermissionChecker = new PermissionChecker(sTree);
   }
 
+  // Helper function to create a path and set the permission to what specified in option.
+  private static void createAndSetPermission(String path, CreateFileOptions option)
+      throws Exception {
+    InodeTree.CreatePathResult result = sTree.createPath(new AlluxioURI(path), option);
+    result.getCreated().get(result.getCreated().size() - 1)
```

Comment:
thanks for the explanation, I guess my question is: if the permission status option is set by the client, should the umask be still applied? I would expect the umask to only be applied if the permission status option is not set and the default permissions are used